integration-tests/src/main/java/org/apache/druid/testing/utils/KinesisAdminClient.java

```diff
@@ -107,6 +111,9 @@ public void deleteStream(String streamName)
   public void updatePartitionCount(String streamName, int newShardCount, boolean blocksUntilStarted)
   {
     int originalShardCount = getStreamPartitionCount(streamName);
+    if (originalShardCount == newShardCount) {
```

Comment:
Seems like a candidate for a test case if there are none for this change.

Reply:
this class is used only for testing.